back/spec/acceptance/resident_native_survey_responses_spec.rb

```diff
@@ -415,7 +418,7 @@
 
             # Verify that the custom field value is saved correctly.
             expect(input.reload.custom_field_values).to eq({
-              'custom_field_name2' => { 'id' => new_idea_file.id, 'name' => file_name }
+              'custom_field_name1' => 'Cat', 'custom_field_name2' => { 'id' => new_idea_file.id, 'name' => file_name }
```

Comment:
@jamesspeake Same questions as in [the comment](https://github.com/CitizenLabDotCo/citizenlab/pull/8373/files#r1677208973) for the previous test.